conda/environments/env.yml

```diff
@@ -16,8 +16,12 @@ dependencies:
   - scipy
   - python=3.8
   - cudatoolkit=11.0
+  - cudatoolkit-dev=11.0
+  - gxx_linux-64=9
```

Comment:
Should we add the `cxx-compiler` package? This will handle the activation of the C++ compiler as well

Do we need the `c-compiler` too?

Reply:
If I remember correctly, Installing `- gxx_linux-64=9` also installed gcc_linux-64=9.

I am not sure how cxx-compiler/c-compiler package works. would it be better to use it?

Reply:
Yeah switching to `cxx-compiler` & `c-compiler` would preferable. They operate the same way as when Conda-Build would use them

Reply:
Updated to use cxx-compiler and c-compiler that install `gxx_linux-64=9`.
Thanks @jakirkham!

Reply:
Thanks Gigon :)